### ISSUE TITLE
Broken on devel

### ISSUE THREAD

**Issue by juancarlospaco** [AndreiRegiani/INim#42]
```nim
$ nimble install inim
Downloading https://github.com/AndreiRegiani/INim using git
  Verifying dependencies for inim@0.4.1
      Info: Dependency on cligen@>= 0.9.15 already satisfied
  Verifying dependencies for cligen@0.9.15
 Installing inim@0.4.1
   Building inim/inim using c backend
    Prompt: Build failed for 'inim@0.4.1', would you like to try installing 'inim@#head' (latest unstable)? [y/N]
    Answer: y
Downloading https://github.com/AndreiRegiani/INim using git
  Verifying dependencies for inim@#head
      Info: Dependency on cligen@>= 0.9.15 already satisfied
  Verifying dependencies for cligen@0.9.15
 Installing inim@#head
   Building inim/inim using c backend
       Tip: 13 messages have been suppressed, use --verbose to show them.
     Error: nimble.nim(1106)         nimble
        ... nimble.nim(1044)         doAction
        ... nimble.nim(497)          install
        ... nimble.nim(483)          install
        ... nimble.nim(359)          installFromDir
        ... nimble.nim(253)          buildFromDir
        ... nimble.nim(245)          buildFromDir
        ... Build failed for package: inim
        ... Details:
        ... nimble.nim(1106)         nimble
        ... nimble.nim(1044)         doAction
        ... nimble.nim(497)          install
        ... nimble.nim(483)          install
        ... nimble.nim(359)          installFromDir
        ... nimble.nim(253)          buildFromDir
        ... nimble.nim(237)          buildFromDir
        ... tools.nim(37)            doCmd
        ... Execution failed with exit code 1
        ... Command: "/home/juan/.nimble/bin/nim" c --noBabelPath -d:release --path:"/home/juan/.nimble/pkgs/cligen-0.9.15"  -o:"/tmp/nimble_7403/githubcom_AndreiRegianiINim_#head/inim" "/tmp/nimble_7403/githubcom_AndreiRegianiINim_#head/src/inim.nim"
        ... Output: Hint: used config file '/home/juan/.choosenim/toolchains/nim-#devel/config/nim.cfg' [Conf]
        ... Hint: system [Processing]
        ... Hint: inim [Processing]
        ... Hint: os [Processing]
        ... Hint: strutils [Processing]
        ... Hint: parseutils [Processing]
        ... Hint: math [Processing]
        ... Hint: bitops [Processing]
        ... Hint: algorithm [Processing]
        ... Hint: unicode [Processing]
        ... Hint: times [Processing]
        ... Hint: options [Processing]
        ... Hint: typetraits [Processing]
        ... Hint: strformat [Processing]
        ... Hint: macros [Processing]
        ... Hint: posix [Processing]
        ... Hint: ospaths [Processing]
        ... Hint: osproc [Processing]
        ... Hint: strtabs [Processing]
        ... Hint: hashes [Processing]
        ... Hint: streams [Processing]
        ... Hint: cpuinfo [Processing]
        ... Hint: linux [Processing]
        ... Hint: rdstdin [Processing]
        ... Hint: linenoise [Processing]
        ... Hint: termios [Processing]
        ... Hint: terminal [Processing]
        ... Hint: colors [Processing]
        ... Hint: tables [Processing]
        ... inim.nim(79, 16) Error: type mismatch: got <nil> but expected 'string'

$ choosenim --version
choosenim v0.3.2 (2018-02-28 14:12:37) [linux/amd64]

$ choosenim show
   Channel: devel
   Version: #devel
      Path: /home/juan/.choosenim/toolchains/nim-#devel

$ uname -a
Linux z 4.18.7-arch1-1-ARCH #1 SMP PREEMPT Sun Sep 9 11:27:58 UTC 2018 x86_64 GNU/Linux

$ nim --version
Nim Compiler Version 0.18.1 [Linux: amd64]
Compiled at 2018-09-14
Copyright (c) 2006-2018 by Andreas Rumpf

active boot switches: -d:release
```

**Comment by Bennyelg** [comment]
same here.

**Comment by Yardanico** [comment]
@AndreiRegiani you can fork cligen and make needed changes for devel until your PR will be accepted in upstream

**Comment by juancarlospaco** [comment]
Another option is drop Cligen, I think most of us just run it without too many additional params anyways...

**Comment by kaushalmodi** [comment]
cligen doesn't need to be dropped. It's been updated to support nim 0.19.0 the day nim 0.19.0 got released.

**Comment by MatthewScholefield** [comment]
Is this still a problem? I just installed inim and even though 0.4.1 failed, devel succeeded.

**Comment by kaushalmodi** [comment]
I can confirm. @AndreiRegiani can you please a new release of inim? `nimble install inim@#head` is installing fine for me (using nim 1.0.2+ (devel)).

**Comment by juancarlospaco** [comment]
Looking at his GitHub profile it seems hes not even coding anything anymore for more than a year.
Wondering if someone want to take over the package and send the PR to nimble packages (?).
:thinking:

**Comment by Yardanico** [comment]
@juancarlospaco what about this
![image](https://user-images.githubusercontent.com/21169548/69013110-8e42c280-098d-11ea-984c-57c7f9ba6981.png)

**Comment by juancarlospaco** [comment]
You are right, whatsoever is a great work I hope it gets fixed.

**Comment by Tangdongle** [comment]
Also fixed with version 0.4.2

**Comment by 0atman** [comment]
YES DUDE
